spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java

```diff
@@ -43,12 +59,40 @@ public Object getProperty(String name) {
 
     public void initProperties() {
         // * for wildcard match
+        // Reading in Configurations
         QueryOptions queryOptions = new QueryOptions().withKeyNames(context + "*").withLabels(label);
         List<KeyValueItem> items = source.getKeys(storeName, queryOptions);
-
         for (KeyValueItem item : items) {
             String key = item.getKey().trim().substring(context.length()).replace('/', '.');
             properties.put(key, item.getValue());
         }
+
+        // Reading In Features
+        List<String> queries = new ArrayList<String>();
+        queries.add("*appconfig*");
+        queryOptions = new QueryOptions().withKeyNames(queries)
+                .withLabels(label);
+        items = source.getKeys(storeName, queryOptions);
+        FeatureSet featureFile = new FeatureSet();
+        for (KeyValueItem item : items) {
+            FeatureManagementItem featureItem;
+            try {
+                featureItem = mapper.readValue(item.getValue(), FeatureManagementItem.class);
+                Feature feature = new Feature();
+                feature.setEnabled(featureItem.getEnabled());
+                feature.setId(featureItem.getId());
+                feature.setEnabledFor(featureItem.getConditions().getClientFilters());
+                featureFile.addFeature(feature);
+            } catch (IOException e) {
+                LOGGER.error("Unabled to parse Feature Management values from Azure.");
```

Comment:
Log the exception `e` to the log? And also do we need to fail here directly? Because user is using this feature management feature, but it failed to load the `FeatureManagementItem`, which may make the feature management feature not working(I suppose), the user needs to know this I think, instead of ignore the exception by silently log the exception.